collections/openapi/schemas/extent.json

```diff
@@ -1,41 +1,70 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Extent Schema",
-    "description": "The extent of the resources in the collection. Only spatial and temporal extents are specified by API-Collections. Additional API standards may add members to represent other extents, for example, thermal or pressure ranges.",
+    "description": "The extent of the resources in the collection. Only spatial and temporal extents are specified. Additional API standards may add members to represent other extents, for example, thermal or pressure ranges.",
     "type": "object",
     "properties": {
         "spatial": {
+            "description": "The spatial extend of the resources in the collection.",
             "type": "object",
             "properties": {
                 "bbox": {
-                    "description": "One or more bounding boxes that describe the spatial extent of the resource collection. Only a single bounding box is supported by API-Collections. Extensions may support additional areas. If multiple areas are provided, the union of the bounding boxes describes the spatial extent.",
+                    "description": "One or more bounding boxes or cubes which describe the spatial extent of the collection. If multiple areas are provided, the union of the bounding boxes describes the spatial extent.",
                     "type": "array",
                     "minItems": 1,
                     "items": {
                         "type": "array",
-                        "minItems": 4,
-                        "maxItems": 6,
+                        "oneOf": [
+                            {
+                                "minItems": 4,
+                                "maxItems": 4
+                            },
+                            {
+                                "minItems": 6,
+                                "maxItems": 6
+                            }
+                        ],
```

Comment:
Wouldn't it be better to have actual schema definitions for each object, eg `BoundingBox2D` and `BoundingBox3D` that defines their min/max and specific description, and use `oneOf` those objects by `$ref` ?